docs/Installation.md

```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
 
 Add a data source:
 
-![S7 Connector Data Source](graphics/S7_Connector_Data_Source.PNG)
+![AdddatasourceS7](graphics/AdddatasourceS7.PNG)
 
 For this example the following tags are added:
 
-![s7_connector_config](graphics/S7_Connector_Configuration.PNG)
+- GDB.signals.tanksSignals.actLevel
+- GDB.signals.tankSignals.actTemprature
+- GDB.process.numberProduced
+- GDB.process.numberFaulty
+
+![S7tags](graphics/S7tags.PNG)
 
 Edit the settings:
 
-![s7_connector_settings](graphics/S7_Connector_Settings.PNG)
+![databussettings](graphics/databussettings.PNG)
 
 Hint: Username and password should be the same for all system apps, e.g. "edge" / "edge".
 
 Deploy and start the project.
 
-## Configure Data Service
+## Configure IIH Essentials
```

Comment:
The Configuration of the IIH Essentials part is a part short, would be great if you can explain it a bit more. You can use the old example as a reference

Reply:
Yeah I added some extra points